dataland-csvconverter/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine:latest
+
+COPY ./CompanyAssociatedEuTaxonomyData.json /storedData/CompanyAssociatedEuTaxonomyData.json
+COPY ./CompanyInformation.json /storedData/CompanyInformation.json
+
+CMD cp /storedData/*.json /testData
```

Comment:
warum haben wir die Daten doppelt?

Reply:
verstanden. Stimmt - hattest du ja erklärt. 
Ich hatte in meinem Kopf CMD mit RUN verwechselt.